src/main/kotlin/no/nav/infotrygd/kontantstotte/service/InnsynService.kt

```diff
@@ -50,9 +49,9 @@ class InnsynService(
         val stønader = stonadRepository.findByOpphoertVfomEquals("000000")
         logger.info("Fant ${stønader.size} stønader")
 
-        val foedselsnumre = stønader.map { stonad -> stonad.barn.map { barn -> barn.fnr.reversert } }
+        val foedselsnumre = stønader.map { stonad -> stonad.barn.map { barn -> barn.fnr } }
         logger.info("Fant ${foedselsnumre.size} foedselsnumre")
-        return foedselsnumre.flatten()
+        return foedselsnumre.flatten().map { it.asString }
     }
 
```

Comment:
Kommentar på linje 47 kan kanskje slettes

Og linje 88 kan man nok nå fjerne tilReversert

barn = this.barn.map { BarnDto(fnr = Foedselsnummer.tilReversert(it.fnr.asString)) },

Reply:
Lurer også på om det må fjernes i `tilFoedselsnummere`-funksjonen i Innsyn.kt (linje 26-28)

Reply:
Vil tro det